armbian/base/scripts/systemd-start-bitcoind.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
(Optional) Should we consider adding some options to make these scripts more robust? I.e `set -euo pipefail`? (And elsewhere.)

Reply:
Good point, I'll do that in the next iteration.